src/lib/model/InfiniteAccessor.ts

```diff
@@ -182,11 +181,11 @@ export class InfiniteAccessor<S, Arg = any, Data = any, E = unknown> extends Acc
         }
         this.currentTask = 'idle';
         this.onFetchingFinish();
-        return data;
+        return error ? ([null, error] as const) : ([data, null] as const);
```

Comment:
It's more common that the error is placed in the first argument, isn't it?

Reply:
In what situations does it make more sense to put error before data? Could you provide an example?

Reply:
I put error before data and make a more typesafe definition in [76c279c](https://github.com/jason89521/daxus/pull/137/commits/76c279ce73cc836898ac4b55884e73c0e32e1141)

Reply:
Like built-in node.js api like `fs.readFile`. If you don't use fs/promise, you can receive a callback which is in form of `function(err, data) {...}`. As for third-party packages, `mongoose` follows the same pattern.

For other languages like golang lack exception-based error handling (try/catch), they usually `return nil, data` or  `return err, nil` when an error occurs.